easybuild/easyconfigs/s/Stacks/Stacks-2.53-iccifort-2019.5.281.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Stacks'
+version = '2.53'
+
+homepage = 'https://catchenlab.life.illinois.edu/stacks'
+description = """Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
+ the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
+ for the purpose of building genetic maps and conducting population genomics and phylogeography.
+"""
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+
+source_urls = ['http://catchenlab.life.illinois.edu/stacks/source/']
```

Comment:
Doesn't this also have an https equivalent...?

Reply:
Ok, changed it to https myself, and asked boegelbot to rebuild with force-download. Seems to work :)